Src/Support/Google.Apis.Auth/OAuth2/ServiceAccountCredential.cs

```diff
@@ -297,7 +297,7 @@ public override async Task<string> GetAccessTokenForRequestAsync(string authUri
                 throw new GoogleApiException(TokenServerUrl, "Invalid OAuth scope or ID token audience provided. " +
                     "A valid authUri and/or OAuth scope is required to proceed.");
             }
-            if (HasExplicitScopes && !UseJwtAccessWithScopes)
+            if (User != null || (HasExplicitScopes && !UseJwtAccessWithScopes))
```

Comment:
Please update the documentation both for this method and the UseJwtAccessWithScopes property.

Reply:
+1